### PR TITLE
[APII-4145] Updated to use PATCH for modification

### PR DESCRIPTION
**Pull request by brijendra98**
- Updates to use PATCH instead of the old PUT endpoint for modification of Client Responses
 - Modified Travis config to use fixed versions of eslint for consistency instead of pulling latest

### CODE REVIEW

**Comment by jwbanning** [comment]
This looks good to me.. This aligns to the documentation on the developer portal. Thanks Brij

File: server/server.js [hunk 1]
```diff
@@ -269,10 +269,10 @@ This endpoint sends a put request to the Client Response API
 for modifying a single Client Response by its Response GUID,
```
Comment (zach) [inline server/server.js]: Please modify the comment on line 268 to say "patch" now.

File: server/server.js [hunk 2]
```diff
@@ -269,10 +269,10 @@ This endpoint sends a put request to the Client Response API
...
 using a client's passkey and OAuth2 access token
 */
-app.put('/api/clientResponses/:responseGuid', setAccessToken, (req, res) => {
+app.patch('/api/clientResponses/:responseGuid', setAccessToken, (req, res) => {
 
   const options = {
-    method: 'put',
+    method: 'patch',
```
Comment (zach) [inline server/server.js]: Please use `PATCH` instead of `patch`.

File: server/server.js (hunk 1 above)
Comment (brijendra98) [reply server/server.js]: Done

File: server/server.js (hunk 2 above)
Comment (brijendra98) [reply server/server.js]: Done

**Pull request merged**